csharp-basics/exercises/Arithmetic/exercise8/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace exercise8
+{
+    class Program
+    {
+        static void CalculateSalary(float basePay, int workedHours)
+        {
+            const int legalLimitHours = 60;
+            const int nonOvertimeHours = 40;
+            const float legalMinWage = 8.0f;
+            const float legalOvertimeModifier = 1.5f;
+            
+                if(basePay< legalMinWage)
+                {
+                    Console.WriteLine("ERROR - The base pay is too little");
+                }
+                else if (workedHours > legalLimitHours)
+                {
+                    Console.WriteLine("ERROR - Too many hours worked");
+                }
+                else
+                {
+                    if (workedHours <= nonOvertimeHours) { Console.WriteLine("$" + (workedHours * basePay)); }
```

Comment:
Kāpēc šis nav vēl vienā else ifā?

Reply:
Ēs neredzu nepieciešamību pēc vēl viena else if... šajā gadījumā pēdējais else var izpildīties tikai tādā gadījumā, ja basePay >= legalMinWage un ja workedHours <= legalLimitHours, nav atšķirības vai tiek atstāts else vai pielikts else if ar šiem nosacījumiem, variants ar else if tikai padara kodu garāku un mazāk sakārtotu manā viedoklī. Bet es varu protams arī pielikts else if pie nepieciešamības